learning-objectives/intl/es.yml

```
@@ -1300,6 +1300,48 @@ typescript/decorators:
     - title: Documentación oficial de Typescript
       url: https://www.typescriptlang.org/docs/handbook/decorators.html
 
+ai-prompting:
+  title: IA Prompting
```

```suggestion
  title: AI Prompting
```

Lo dejaría como "AI Prompting" sólo para que no esté esa mezcla entre inglés y español, preferiría a que esté o todo en español o todo en inglés, pero la mezcla se siente rara